src/proto/extension.rs

```diff
@@ -32,6 +34,14 @@ impl Decode for SessionBind {
     }
 }
 
+/// restrict-destination-v00@openssh.com key constraint extension
```

Comment:
I think backticks around the constraint extension name would be a good idea:

```suggestion
/// `restrict-destination-v00@openssh.com` key constraint extension
```

Reply:
agreed; will update them now